.github/workflows/ci.yml

```
@@ -23,6 +23,10 @@ on:
         required: true
         default: 'default'
         type: string
+      wc-version:
+        description: 'Specific WooCommerce version to test against.'
```

Not all test jobs are using this, it can be confusing

```suggestion
        description: 'Specific WooCommerce version to install by the test jobs supporting it'
```